packages/opentelemetry-sdk-trace-base/src/export/ConsoleSpanExporter.ts

```diff
@@ -45,6 +45,14 @@ export class ConsoleSpanExporter implements SpanExporter {
    * Shutdown the exporter.
    */
   shutdown(): Promise<void> {
+    return this._flush();
+  }
+
+  forceFlush(): Promise<void> {
```

Comment:
Why export an empty array?

Reply:
I'm not sure, this is what the shutdown method was originally doing, and I kept the same implementation for forceFlush, you think this should be removed?

Reply:
It's fine if it was already there